tools/docker/runner/entrypoint.sh

```diff
@@ -24,6 +25,7 @@ run ip link set      dev wlan2    master br-lan
 run ip address flush dev eth0
 run ip link set      dev wlan0    up
 run ip link set      dev wlan2    up
+run ip link set      dev br-lan   addr "$bridge_mac"
```

Comment:
For the Turris Omnia, we have this in the wiki:
```
The backhaul interface must have a different MAC address than the bridge interface. 
```

Does it not apply to the dummy variant?

Reply:
That is an extremely good observation! I missed the fact that originally we had a fixed MAC address that was different from eth0.

After you removed the explicit setting of the addresses, br-lan receives the same address as eth0 in CI:
```
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN mode DEFAULT group default qlen 1
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
2: br-lan: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP mode DEFAULT group default qlen 1000
    link/ether 02:42:ac:13:00:03 brd ff:ff:ff:ff:ff:ff
3: wlan0: <BROADCAST,NOARP,UP,LOWER_UP> mtu 1500 qdisc noqueue master br-lan state UNKNOWN mode DEFAULT group default qlen 1000
    link/ether d6:82:42:41:62:97 brd ff:ff:ff:ff:ff:ff
4: wlan2: <BROADCAST,NOARP,UP,LOWER_UP> mtu 1500 qdisc noqueue master br-lan state UNKNOWN mode DEFAULT group default qlen 1000
    link/ether c2:39:82:4b:44:b1 brd ff:ff:ff:ff:ff:ff
188015: eth0@if188016: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue master br-lan state UP mode DEFAULT group default 
    link/ether 02:42:ac:13:00:03 brd ff:ff:ff:ff:ff:ff link-netnsid 0
```

Apparently, it now works by accident... I think it might have something to do with the way that docker generates MAC addresses: they are sequential, so repeater1's address is the gateway address + 1. This happens to be the same as the fake address that is generated for gateway's ETHERNET node, and apparently the two problems compensate for each other...

But this might very well explain why test_flows still doesn't work for me. I'll look further into it.